ghcide/src/Development/IDE/GHC/Compat/Driver.hs

```diff
@@ -0,0 +1,125 @@
+-- This module copies parts of the driver code in GHC.Main.Driver to provide
```

Comment:
Make an upstream issue to expose this and link to it here so we know when we can get rid of this

Reply:
This was a TODO "Make PR on GHC side so that we can eventually remove ghcide/src/Development/IDE/GHC/Compat/Driver.hs" - done now :)

Issue https://gitlab.haskell.org/ghc/ghc/-/issues/24996 and MR https://gitlab.haskell.org/ghc/ghc/-/merge_requests/12891

Reply:
Added to the source code as well in 3500ac3e